deebotozmo/constants.py

```diff
@@ -26,6 +26,9 @@
 COMPONENT_MAIN_BRUSH = 'brush'
 COMPONENT_FILTER = 'heap'
 
+SCHEDULE_ADD = 'add'
```

Comment:
You added constant are not used? Did you forget to push some code?
If not please remove unused code